lib/tasks/deployment/20240624130949_complete_started_already_completed_programmes.rake

```diff
@@ -0,0 +1,21 @@
+namespace :after_party do
+  desc 'Deployment task: complete_started_already_completed_programmes'
+  task complete_started_already_completed_programmes: :environment do
+    puts "Running deploy task 'complete_started_already_completed_programmes'"
+
+    Programme.started.each do |programme|
+      if programme.all_activities_complete?
+        puts "Setting programme '#{programme.programme_type.title}' to complete for school: #{programme.school.name}"
+        programme.ended_on = programme.started_on
```

Comment:
Would it not be better to end them on the date this task is run?

Reply:
Set this intentionally as the start date, given the date they started the programme will have been the date they finished it. Happy to change though?

Reply:
OK, I guess as you're plugging a loop-hole it makes sense. So leave it as it is